build.psm1

```diff
@@ -197,7 +197,7 @@ function Start-PSBuild {
         $Arguments += "build"
     }
     if ($Output) {
-        $Arguments += "--output", (Join-Path $PSScriptRoot $Output)
+        $Arguments += "--output", $Output
```

Comment:
I don't see any reason a relative path would fail here; it's sent to `dotnet build --output /the/path` and that can take relative paths.

Adding the `Join-Path $PSScriptRoot` was my mistake previously; I think for some reason I wanted all output to be relative to the Git repo, but that's totally useless.